src/components/LeftSideBar.js

```diff
@@ -0,0 +1,141 @@
+import styled from "styled-components";
+
+const LeftSideBar = (props) => {
+  return (
+    <Container>
+      <Card>
+        <UserInfo>
+          <CardBackground />
+          <a>
+            <Photo />
+            <Link>Welcome, there!</Link>
+          </a>
+          <a>
+            <AddPhotoText>Add a photo</AddPhotoText>
+          </a>
+        </UserInfo>
+        <Widget>
+          <a>
+            <div>
+              <span>Connections</span>
+              <span>Grow your Network</span>
+            </div>
+            <img src="./images/widget-icon.svg" alt="" />
```

Comment:
- Place all image sources on a separate constant files. So that they can be resuable and centered.
- Please add the alt texts